pkg/generators/forwarding/fluentd/fluent_conf_test.go

```diff
@@ -2718,7 +2718,7 @@ var _ = Describe("Generating fluentd config", func() {
               @id audit-input
               @label @MEASURE
               path "#{ENV['AUDIT_FILE'] || '/var/log/audit/audit.log'}"
-              pos_file "#{ENV['AUDIT_POS_FILE'] || '/var/log/audit/audit.log.pos'}"
+              pos_file "#{ENV['AUDIT_POS_FILE'] || '/var/lib/fluentd/pos/audit/audit.log.pos'}"
```

Comment:
Presumably this will fail start given "audit" is not a directory under "pos".  Modify to remove the env var as well as the directory "audit"

Reply:
fluentd will create /var/lib/fluentd/pos/audit, it does a `mkdir -p` on the path of the pos file: https://github.com/openshift/origin-aggregated-logging/blob/c4a31147e4af9883d2ad749f4060a3c70e641816/fluentd/vendored_gem_src/fluentd/lib/fluent/plugin/in_tail.rb#L197.
I didn't quite get the second part, do we not need the AUDIT_POS_FILE env var anymore?

Reply:
No

Reply:
Remove the audit sub directory and rename file if needed to avoid naming conflicts

Reply:
Done